colonyzer2/functions.py

```diff
@@ -32,11 +32,11 @@ def readInstructions(fullpath,fname='Colonyzer.txt',searchUpStream=False):
             if fullpathnew!=fullpath:
                 fullpath=fullpathnew
             else:
-                raise(ValueError("Searched up to "+fullpath+" but "+fname+" not found in directory structure."))
+                raise ValueError
```

Comment:
Need to keep the message here, though I can't remember why I wrote "raise" as a function call.  It seems that in Python 3 it should be:

raise ValueError("Searched up to "+fullpath+" but "+fname+" not found in directory structure.")